Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -26,6 +26,10 @@ public class GazeProvider :
 
         private const float MovementThreshold = 0.01f;
 
+        [SerializeField]
+        [Tooltip("If true, the gaze cursor will stay locked on the object when the pointer's focus is locked, otherwise it will continue following the head's direction")]
+        private bool lockCursorWhenFocusLocked = false;
```

Comment:
Re: @cre8ivepark's comment

```suggestion
        private bool lockCursorWhenFocusLocked = true;
```